index.html

```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
             </div>
         </div>
     </section> 
+    <br>
+    <br>
+    <br>
+    <section class="tonic">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
+        </div>
+        <ul class="toniclists1">
+            <li class="canopy">
```

Comment:
- [ ] Kindly take note that you are to wrap all the cards section in `main` tag and display it as grid. 😄 
![crr5](https://user-images.githubusercontent.com/103386183/198280225-67893911-e05f-4f73-a8c6-0762de7aafb1.png)

Reply:
## resolved 
- I did not follow instructions to fix everything in the grid 
- I had to remove the <br> and use the grid for everything
- Issue resolved and it's even looking better 
- Thanks again